packages/core/src/index.ts

```diff
@@ -121,7 +122,7 @@ export default class APICore {
         init.signal = controller.signal;
       }
 
-      return fetchHar(har as any, {
+      return fetchHar(har as Har, {
```

Comment:
The HAR coming out of `@readme/oas-to-har` is a HAR but not a _full_ one so we need to cast this.